Sources/SwiftSyntaxBuilderGeneration/Child.swift

```diff
@@ -95,14 +96,18 @@ class Child {
     self.requiresLeadingNewline = requiresLeadingNewline
     self.isOptional = isOptional
 
-    var mappedTokenChoices = [Token]()
+    let isToken = syntaxKind.hasSuffix("Token")
+    var mappedTokenChoices = [Token?]()
     
-    if syntaxKind.hasSuffix("Token"), let token = SYNTAX_TOKEN_MAP[syntaxKind] {
+    if isToken, let token = SYNTAX_TOKEN_MAP[syntaxKind] {
       mappedTokenChoices.append(token)
     }
 
-    mappedTokenChoices.append(contentsOf: tokenChoices.compactMap { SYNTAX_TOKEN_MAP[$0] })
-    self.tokenChoices = mappedTokenChoices
+    mappedTokenChoices.append(contentsOf: tokenChoices.map { SYNTAX_TOKEN_MAP["\($0)Token"] })
```

Comment:
This fixes a bug where `self.tokenChoices` was always empty since the `tokenChoices` do not have the `Token` suffix.